API/controllers/entityController.go

```diff
@@ -454,8 +454,7 @@ func HandleGenericObjectWildcard(w http.ResponseWriter, r *http.Request) {
 				u.RespondWithError(w, modelErr)
 			}
 		}
-		w.WriteHeader(http.StatusNoContent)
-		u.Respond(w, u.Message("successfully deleted"))
+		u.Respond(w, u.RespDataWrapper("successfully deleted objects", matchingObjects))
```

Comment:
This endpoint no longer exists with the refacto, so I copied this response to the delete of /api/objects/id in the branch 229.